api/metrics-api-jaxrs/src/main/webapp/static/css/main.css

```diff
@@ -1,3 +1,15 @@
+.navbar-fixed-top {
```

Comment:
@gbaufake yes I've seen that. Will update it later (currently replacing react to angular, so anyway I'll have to regenerate all licenses)